android/src/main/kotlin/com/bitmovin/player/flutter/FlutterPlayerView.kt

```diff
@@ -82,4 +110,11 @@ class FlutterPlayerView(
     override fun onCancel(arguments: Any?) {
         sink = null
     }
+
+    private fun Context.getActivity(): Activity? =
+        when (this) {
+            is Activity -> this
+            is ContextWrapper -> baseContext.getActivity()
+            else -> null
```

Comment:
throw instead of returning null? this could even be inlined.

Reply:
I went for nullable approach to be able to provide a more descriptive error message where it's used (FlutterPlayerView.kt:46). 
Since we might also need it somewhere else I decided to just leave it generic, so that it can just be extracted in that case.